spring-ai-bedrock/README.md

```diff
@@ -0,0 +1,83 @@
+# Bedrock AI Chat and Embedding Clients
```

Comment:
The documentation for these base clients still needs to go into our reference documentation, which is in adoc format.  I will create a separate issue for that.